docs/contributor/arch/002-fluent-bit-without-file-system-buffer.md

```diff
@@ -0,0 +1,32 @@
+
+The persistent buffer can prevent log loss only for a short period of time. Depending on the amount of generated logs, this is usually in the range of minutes to a few hours and might be too short to restore an outage of a log backend or detect and solve a faulty configuration.
+
+After the in-cluster Loki backend has been removed, the pipeline isolation requirements must be reevaluated in favor of pausing the input plugin. The typical setup are clusters with a single LogPipeline or multiple LogPipelines that ingest logs to the same backend; for example, ingesting application logs and Istio access logs into two different indexes of an [SAP Cloud Logging](../../user/integration/sap-cloud-logging) instance. With this typical setup, the benefit of pausing the `tail` input plugin exceeds the pipeline isolation requirements. Kubernetes' [logging architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/) helps by storing logs persistently on the node file-system and rotating them automatically after reaching a certain file size.
+
+## Decision
+
+Fluent Bit can be configured in different ways to read container logs and ingest them into multiple backends:
+
+1. **Single log stream:** Logs are read by a `tail` input plugin, one or more filters are applied, and written to the backend by multiple output plugins. The input cannot be controlled based on the condition of individual outputs. Filters cannot be applied per individual pipeline. The complexity of this setup is low.
+2. **Split log stream**: Logs are read by a `tail` input plugin. Before the stream is split into a pipeline-specific stream, global filters like the Kubernetes filter are applied; and before logs are written to an output, each stream can have additional filters. Pipelines can be isolated using the persistent buffer. However, it's impossible to pause the input in the situation of a backend outage. The complexity of this setup is high.
+3. **Dedicated log streams**: Each pipeline has its own `tail` input plugin, a list of filters, and an output plugin. This setup isolates the log processing between different pipelines. In the case of a problem, the streams can be paused individually. This setup has medium complexity.
+4. **Dedicated Fluent Bit instances**: Each pipeline gets its own Fluent Bit DaemonSet. This setup isolates also the CPU and memory resources per pipeline with the cost of a larger overhead. This setup has medium complexity.
+
+Option 1 does not fulfill our requirement to apply individual filters per pipeline. Option 4 causes an unacceptable resource overhead for our typical setup of two pipelines (application logs and access logs). Option 2 and 3 allow log filter settings per pipeline. Currently, Telemetry Manager uses option 2.
```

Comment:
Seeing it in context, this might be better:

```suggestion
Currently, Telemetry Manager uses option 2. 
Option 1 does not fulfill our requirement to apply individual filters per pipeline. Option 4 causes an unacceptable resource overhead for our typical setup of two pipelines (application logs and access logs). Option 2 and 3 allow log filter settings per pipeline.
```